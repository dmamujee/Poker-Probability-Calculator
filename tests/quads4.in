6 H
12 H
5 H
13 S
13 C
13 H
13 D