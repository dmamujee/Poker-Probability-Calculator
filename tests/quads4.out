Hand Ranking: Four of a Kind
